db/migrate/20240404182018_create_new_org_and_rep_tables.rb

```diff
@@ -0,0 +1,85 @@
+class CreateNewOrgAndRepTables < ActiveRecord::Migration[7.1]
+  # rubocop:disable Metrics/MethodLength
+  def change
+    create_table :accredited_representatives, id: false do |t|
+      t.string :registration_number, null: false
+      t.string :poa_code
```

Comment:
`accredited_representatives` shouldn't have the `poa_code` column. That will be handled by the join table.

Reply:
That field is for the accredited rep or attorney's POA, not the VCO.

Reply:
Ok, it works for attorneys and claim agents, but not veteran service officers with many poa code associations. Will this be nil for veteran service officers?